### PR TITLE
Optimizes client module loading and blocks dump import.

### PR DESCRIPTION
**Pull request by eugene-shevchenko**
Improves ethereumj client launching speed. 
Optimizes blocks dump import workflow.

### CODE REVIEW

**Comment by holiman** [comment]
I see there are only changes for `develop` -- we should either remove `master` or keep it up to date, imo

File: clients/ethereumj:develop/harmony.sh [hunk 1]
```diff
@@ -99,16 +103,14 @@ echo "Loading initial blockchain..."
...
 
 # Load the remainder of the test chain
 if [ -d /blocks ]; then
     echo "Loading remaining individual blocks..."
-    for block in `ls /blocks | sort -n`; do
-        export HARMONY_ETHER_CAMP_OPTS="$FLAGS -Dblocks.format=rlp -Dblocks.loader=/blocks/$block -Dpeer.listen.port=0"
-		eval "${HARMONY_EXECUTABLE} importBlocks"
-	done
+    export HARMONY_ETHER_CAMP_OPTS="$FLAGS -Dblocks.format=rlp -Dblocks.loader=/blocks -Dpeer.listen.port=0"
+    eval ${HARMONY_EXECUTABLE}
```
Comment (holiman) [inline clients/ethereumj:develop/harmony.sh]: I've seen this before, and am a bit curious. What's the benefit in using `eval` for executing a command? None of the other client-script uses that.

Comment (eugene-shevchenko) [reply clients/ethereumj:develop/harmony.sh]: It's a deprecated part of code. I've replaced this command with `run_harmony` function already.

**Comment by eugene-shevchenko** [comment]
@holiman hi, does this review need any extra changes?

**Pull request merged**